examples/cluster/plot_face_ward_segmentation.py

```diff
@@ -23,20 +23,14 @@
 
 from sklearn.feature_extraction.image import grid_to_graph
 from sklearn.cluster import AgglomerativeClustering
-from sklearn.utils.testing import SkipTest
-from sklearn.utils.fixes import sp_version
-
-if sp_version < (0, 12):
-    raise SkipTest("Skipping because SciPy version earlier than 0.12.0 and "
-                   "thus does not include the scipy.misc.face() image.")
 
 
 ###############################################################################
 # Generate data
 try:
     face = sp.face(gray=True)
 except AttributeError:
-    # Newer versions of scipy have face in misc
+    # SciPy 0.16 and higher have face in misc
```

Comment:
A bit of a nitpick but here and in other places where this code has been copied and pasted, I would do it the other way around:

```py
try:
    # code that works with recent scipy versions
except AttributeError:
    # fall-back for old scipy versions
```